flow/src/main/java/com/sequenceiq/flow/controller/FlowPublicController.java

```diff
@@ -27,4 +30,16 @@ public FlowCheckResponse hasFlowRunningByChainId(String chainId, String resource
     public FlowCheckResponse hasFlowRunningByFlowId(String flowId, String resourceCrn) {
         return flowService.getFlowState(flowId);
     }
+
+    @Override
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)
```

Comment:
I think it is ok now, but the question is why power user? I guess a proper authz would be desirable.

E.g if I am just a DataHub admin, then my permission should be sufficient to check what is happening with my DH.

Reply:
seems like we would need individual endpoints for ever resource, so probably for now it will be available on flow api as well, but if we would want any UI or resource specific features around it, specific endpoints will need to be implemented

Reply:
ok, aadded new endpoints